webapp/src/components/SingleStudy/ExportFilterModal/MultipleSelect.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
could this be factored with the SingleSelect so that it's just a component with a parameter "multiple" ?